homeassistant/components/arcam_fmj/media_player.py

```diff
@@ -9,17 +9,15 @@
 
 from homeassistant.components.media_player import (
     BrowseMedia,
+    MediaClass,
```

Comment:
`MediaClass` should be imported in `media_player.__init__`. That will fix the issue.

Reply:
I was going to rebase over the cast PR. Since that PR is delayed I have added it here.